torch/_inductor/codegen/cpp.py

```diff
@@ -2842,12 +2851,23 @@ def codegen_nodes(self, nodes):
 
         kernel_group.finalize_kernel(cpp_kernel_proxy, nodes)
 
+        args_num = self.get_num_args()
+        if args_num > CppScheduling.MAX_FUSED_KERNEL_ARGS_NUM:
+            self._set_flush_status(True)
+
+    def get_num_args(self):
+        return self.kernel_group.get_num_args()
```

Comment:
Is this used anywhere?

Reply:
> Is this used anywhere?

it used at cpp backend codegen_node for args check: https://github.com/pytorch/pytorch/pull/113131/files#diff-5ab7b0235e2076a5fc6629ba0b109208940f5b94f5c13babc3e0f87cf4fcec82R2854

Reply:
@mlazos if rename it to `_get_scheduled_num_args`? It seems better.

Reply:
> @mlazos if rename it to `_get_scheduled_num_args`? It seems better.

renamed.

Reply:
Got it

Reply:
@mlazos whether I can merge it?